mifosng-android/src/main/java/com/mifos/mifosxdroid/AboutActivity.java

```diff
@@ -0,0 +1,44 @@
+package com.mifos.mifosxdroid;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.view.View;
+import androidx.annotation.Nullable;
+import com.mifos.mifosxdroid.core.MifosBaseActivity;
+
+public class AboutActivity extends MifosBaseActivity {
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_about);
+        showBackButton();
+    }
+
+    public void goToweb (View view) {
+        String url = "https://github.com/openMF/android-client/graphs/contributors";
+        link(url);
+    }
+
+    public void goToGit (View view) {
+        String url = "https://github.com/openMF/android-client";
+        link(url);
+    }
+
+    public void goToTwitter (View view) {
+        String url = "https://twitter.com/mifos";
+        link(url);
```

Comment:
```suggestion 
        link("https://twitter.com/mifos");

Reply:
Remove this variable, it's useless, do this for every other function.

Reply:
Ok